pkg/kubelet/cm/cgroup_manager_linux.go

```diff
@@ -514,26 +499,31 @@ func setResourcesV2(cgroupConfig *libcontainerconfigs.Cgroup) error {
 }
 
 func (m *cgroupManagerImpl) toResources(resourceConfig *ResourceConfig) *libcontainerconfigs.Resources {
-	resources := &libcontainerconfigs.Resources{}
+	resources := &libcontainerconfigs.Resources{
+		Devices: []*libcontainerconfigs.DeviceRule{
+			{
+				Type:        'a',
+				Permissions: "rwm",
+				Allow:       true,
+				Minor:       libcontainerconfigs.Wildcard,
+				Major:       libcontainerconfigs.Wildcard,
+			},
+		},
+	}
 	if resourceConfig == nil {
 		return resources
 	}
 	if resourceConfig.Memory != nil {
 		resources.Memory = *resourceConfig.Memory
 	}
-	if libcontainercgroups.IsCgroup2UnifiedMode() {
-		resources.CpuWeight = getCpuWeight(resourceConfig.CpuShares)
```

Comment:
why is this removed?  can you provide some context?

Reply:
I think I've done a mistake here.  libcontainer dropped CpuMax but CpuWeight is still needed and we need to do the conversion.  Thanks for noticing it, I'll fix it.